### ISSUE TITLE
Instance @ https://barex.doxr.hackclub.app/ doesn't work

### ISSUE THREAD

**Issue by ascpixi** [DoxrGitHub/hs-yt#2]
Hiya!
Not sure if you're aware, but just in case, I thought to let you know that the URL you provided in your Wonderdome submission - https://barex.doxr.hackclub.app/ - isn't active, and is returning HTTP 502. I tried accessing the website multiple times today to no avail.

**Comment by DoxrGitHub** [comment]
Yeah, sorry. nest hosting goes down really frequently for me. I would try to provide a more reliable form of hosting but I can't easily set that up for NodeJS projects like this. I may try to use render soon.

Edit:

![image](https://github.com/user-attachments/assets/327b19f5-275d-43ab-a9c0-af37d4c22a3b)

@ascpixi Render is not an option, so I simply can't unless I want to attend the project on Nest extremely frequently. I'm sure Nest isn't supposed to be like this but wouldn't it give some other error vs the 502 I get from the barex domain. Anyways, I'm willing to get this up if possible.

**Comment by ascpixi** [comment]
You _could_ try to do some refactoring to make the app serverless-friendly, and then host it on Vercel. You don't have to verify with a credit card on there.
